invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -164,7 +164,7 @@ class UIRecordSchema(BaseObjectSchema):
 
     # Custom fields
     # TODO: take into account vocabularies and localization of titles
-    custom = fields.Nested(
+    custom_fields = fields.Nested(
```

Comment:
This works with the component with setattr?

Reply:
Here we don't have the record passed but the record dictionary as it's on the serializer level... so `NestedAttribute` doesn't work...